std/src/main/scala/algebra/std/boolean.scala

```diff
@@ -9,6 +10,13 @@ package object boolean extends BooleanInstances
 trait BooleanInstances {
   implicit val booleanAlgebra: BooleanAlgebra =
     new BooleanAlgebra
+
+  val BooleanRing = new BoolRing[Boolean] {
```

Comment:
this is actually a subset of the `Fields[Boolean]`. Is there any relationship as to when a `Field[T]` is an extension of a `BoolRing[T]`?